hack/verify-test-featuregates.sh

```diff
@@ -26,7 +26,7 @@ cd "${KUBE_ROOT}"
 rc=0
 
 # find test files accessing the mutable global feature gate or interface
-direct_sets=$(grep -n --include '*_test.go' -R 'MutableFeatureGate' . 2>/dev/null) || true
+direct_sets=$(grep -n --include '.*_test.go' -R 'MutableFeatureGate' . 2>/dev/null) || true
```

Comment:
ref: https://github.com/koalaman/shellcheck/wiki/SC2063

Reply:
I think that it cannot use regex with `--include`. If `.*_test.go`, `grep` does not include any files. See #76855. I proposed another way to fix this line in #80505.

Reply:
thinks.